src/Makefile

```diff
@@ -86,7 +86,7 @@ composer_download:
 composer_install:
 	@echo "current dir: '$${PWD}'"
 	$(INSTALL_DATA) composer.json composer.lock $(DESTDIR)$(MODDIR)
-	(cd $(DESTDIR)$(MODDIR); $(COMPOSERTMP)/composer install -q --no-dev)
+	(cd $(DESTDIR)$(MODDIR); export COMPOSER_HOME=/dev/null ; $(COMPOSERTMP)/composer install -q --no-dev)
```

Comment:
I think that this will not work, since `TOP=..` and thus this line evaluates to 
```
(cd some/other/folder; export ....; ../tmp/composer/composer install -q --no-dev)
```
which does not work.

Setting `COMPOSER_HOME` is a good idea.